### DISCUSSION TITLE
Event Start Time

### DISCUSSION THREAD

**Discussion by VincentClement1** [vasqued2/ha-teamtracker-card#47] [General]
When I view the dashboard that contains the ha-teamtracker card via HA Cloud, the HA Android app or locally via my desktop computer, the event start time is displayed using AM/PM. 

Showing that dashboard via Google Cast to a TV connected via Chromecast Ultra, the event time is displayed in 24 hour time. The Time Format under my profile is set 12 hours (AM/PM) in both Home Assistant and the HA App. 

No other card that shows a time in my dashboard exhibits this behaviour. It's only the ha-teamtracker-card that shows 24 hour via Google Cast. It could be an issue with the "Google Cast: Show lovelace view" service. 

Version | core-2023.2.5
Installation Type | Home Assistant Container
Development | false
Supervisor | false
Docker | true
User | root
Virtual Environment | false
Python Version | 3.10.7
Operating System Family | Linux
Operating System Version | 5.15.84-v7l+
CPU Architecture | armv7l
Timezone | America/New_York

**Comment by vasqued2** [comment]
I don’t have a Google Cast so I can’t recreate or know the details of what you are doing.

The time format can vary for each front end.  It is determined first by the time format selected in your Profile (under your name) and second by the Language selected in your Profile ( English US is AM/PM and English GB is 24) for that device.  If you didn’t set either one, it attempts to set it based on your browser locale.

Try manually changing the language and then setting it back to what you want or doing the same with the time format and see if that resolves it.  You have to do it on whatever device you want it to change.  It’s a local setting.  It won’t use the same for each device.  If you still have issues after that, lmk and we will try some other things.

Changes made in the profile should happen immediately.  No need to reboot or anything.
